documentation/latest/operations/data_synchronization.md

```diff
@@ -183,4 +196,4 @@ In this case on-device `/export/stats` directory will be synced to a
 The Flotta agent synchronizes paths specified in the configuration every 15
 seconds. Only new or changed files are transferred.
 
-Files removed on the device are not removed from the storage.
+Files removed on the device are not removed from the storage. Deleting the workload will remove the used storage.
```

Comment:
The storage on the device will not be removed when workload is removed, only when the device is unregistered.

Reply:
oh odd, so if another workload is added with the same name, it will have access to the data from the previous workload? that could lead to data being leaked from the device to a remote storage. Also, this causes a serious issue for free disk leak since there is no way to cleanup the space used by removed workloads.

I would suggest to allow workloads to define if they want to remove their used storage when they are removed from the device. What do you think?

Reply:
Updated to reflect the unregistered by the way.

Reply:
It is a good idea to allow users to specify data management strategy. 
Workload on device update will have to be refactored/changed - when workload is deployed, the old version is removed first and we don't want to remove the files when that happens.